pkg/cmd/extension/command.go

```diff
@@ -72,6 +85,28 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 					and adding qualifiers as needed. See 'gh help search repos' to learn
 					more about repository search.
 				`),
+				Example: heredoc.Doc(`
+					# List the first 30 extensions sorted by star count, descending
+					$ gh ext search
+
+					# List more extensions
+					$ gh ext search -L300
```

Comment:
You mention `—limit` above but the use `-L` here. Wondering if it’d be useful to do the example with both here.